content/api/resources/trades/list.mdx

```diff
@@ -242,4 +252,15 @@ var es = server
   .stream({ onmessage: tradesHandler });
 ```
 
+```python
+from stellar_sdk import Server, Asset
+server = Server("https://horizon.stellar.org")
+  
+def callback(resp):
+  print(resp)
+  
+for tx in server.trades().for_asset_pair(Asset("USD", "GDUKMGUGDZQK6YHYA5Z6AY2G4XDSZPSZ3SW5UN3ARVMO6QSRDWP5YLEX"), Asset.native()).cursor("now").stream():
```

Comment:
Like before, these line lengths are really clunky. Please try to stick to 80 (or at most 100) chars each:

```suggestion
for tx in server.trades().for_asset_pair(
    Asset("USD", "GDUKMGUGDZQK6YHYA5Z6AY2G4XDSZPSZ3SW5UN3ARVMO6QSRDWP5YLEX"),
    Asset.native()
).cursor("now").stream():
```